### ISSUE TITLE
Support for POST/DELETE methods?

### ISSUE THREAD

**Issue by akennedy4155** [browniebroke/deezer-python#327]
From what I can see in this library there is no support for POST/DELETE.  Only GET methods are supported.  Is there something I'm missing or is that something that could be implemented by me in a PR?

**Comment by browniebroke** [comment]
Yes, that's pretty much accurrate, see also #310. PR to fix that would be welcome 😄

**Comment by akennedy4155** [comment]
@browniebroke Already started working on this, starting with the methods that I needed for my project.  I'm going to scope out the rest of the methods and I'll keep you updated on progress.

**Comment by akennedy4155** [comment]
@browniebroke Letting you know this Issue is __In Progress__.

**Comment by browniebroke** [comment]
I see your progress on your fork, nice one @akennedy4155! Feel free to open a draft PR already to get early feedback. 

Maybe you could send [the first commit](https://github.com/browniebroke/deezer-python/commit/ceb857dff574c51708b6c5a7e7d6b556280e8cbb) in a separate PR that I could merge early on? I'm suggesting this as GitHub actions are restricted for new contributors, and I would have to approve the CI each time you push until you've contributed before.
